src/main/java/com/github/monster860/fastdmm/FastDMM.java

```diff
@@ -805,4 +870,59 @@ private void loop() {
 			Display.sync(60);
 		}
 	}
+
+	private String getAppData() {
```

Comment:
`user.home` resolves to your home directory `c:/users/username` there is no need to use `appdata` because `user.home` just works.